connectivity/suite.go

```diff
@@ -63,8 +63,6 @@ func Run(ctx context.Context, ct *check.ConnectivityTest) error {
 			tests.PodToPod(""),
 			tests.ClientToClient(""),
 			tests.PodToService(""),
-			tests.PodToRemoteNodePort(""),
```

Comment:
I wonder if we should comment these out with the corresponding context so that we don't try to add these back in future until https://github.com/cilium/cilium/issues/17144 is resolved.

The commit message doesn't quite capture the context for why we are disabling the test. At the very least, mention about how nodeport is typically used for N-S traffic, and it's okay to skip the pod-nodeport case as a trade-off.

Reply:
Good point, updated.